kayak_core/src/color.rs

```diff
@@ -19,30 +19,27 @@ impl Default for Color {
 
 impl Color {
     /// The black color.
-    pub const BLACK: Color = Color {
-        r: 0.0,
-        g: 0.0,
-        b: 0.0,
-        a: 1.0,
-    };
+    pub const BLACK: Color = Color::rgb(0.0, 0.0, 0.0);
 
     /// The white color.
-    pub const WHITE: Color = Color {
-        r: 1.0,
-        g: 1.0,
-        b: 1.0,
-        a: 1.0,
-    };
+    pub const WHITE: Color = Color::rgb(1.0, 1.0, 1.0);
+
+    pub const GRAY: Color = Color::rgb(0.5, 0.5, 0.5);
+    pub const RED: Color = Color::rgb(1.0, 0.0, 0.0);
+    pub const GREEN: Color = Color::rgb(0.0, 1.0, 0.0);
+    pub const BLUE: Color = Color::rgb(0.0, 0.0, 1.0);
+    pub const YELLOW: Color = Color::rgb(1.0, 1.0, 0.0);
+    pub const MAGENTA: Color = Color::rgb(1.0, 0.0, 1.0);
+    pub const CYAN: Color = Color::rgb(0.0, 1.0, 1.0);
 
     /// A color with no opacity.
-    pub const TRANSPARENT: Color = Color {
-        r: 0.0,
-        g: 0.0,
-        b: 0.0,
-        a: 0.0,
-    };
+    pub const TRANSPARENT: Color = Color::new(0.0, 0.0, 0.0, 0.0);
 
-    pub fn new(r: f32, g: f32, b: f32, a: f32) -> Self {
+    pub const fn new(r: f32, g: f32, b: f32, a: f32) -> Self {
```

Comment:
Would it be worth renaming this to `rgba` for clarity?

Reply:
Yes, but I can't build this on main, I can only build on the update-bevy branch and I didn't want to do a refactor that would affect other files.

Reply:
Oh strange. Is it a Kayak issue or your local system? Have you tried cleaning and rebuilding?

Reply:
I'm not sure honestly. I tried `cargo clean && rm Cargo.lock` many times and then building with both nightly and stable. I always get 
```bash
   Compiling bevy_render v0.5.0 (https://github.com/bevyengine/bevy?rev=081350916c39fc1c7c9d659ccb6d86413832b465#08135091)
error: cannot find derive macro `Component` in this scope
  --> C:\Users\<username>\.cargo\git\checkouts\bevy-f7ffde730c324c74\0813509\crates\bevy_render\src\render_resource\texture.rs:12:24
   |
12 | #[derive(Clone, Debug, Component)]
   |                        ^^^^^^^^^
   |
   = note: consider importing one of these items:
           bevy_ecs::prelude::Component
           crate::Component

error: could not compile `bevy_render` due to previous error
```

Reply:
But with the update-bevy branch I can build as long as I use stable

Reply:
Do you also need to remove `C:\Users\<username>\.cargo\git\checkouts\bevy-f7ffde730c324c74/...`?

It seems to build fine on my end.

Reply:
So, I deleted the all the `.cargo\git\checkouts\bevy-*` and it works now. I genuinely don't understand what happened. It's clearly a build artifact from cargo, why would it use the wrong one though.

Reply:
I wish I knew 😅

Reply:
So, now that I can compile I did the change and it still compiles